src/app_service/common/wallet_constants.nim

```diff
@@ -12,3 +12,4 @@ const
 
   PARASWAP_V5_APPROVE_CONTRACT_ADDRESS* = "0x216B4B4Ba9F3e719726886d34a177484278Bfcae" # Same address for all supported chains
```

Comment:
@dlipicar do we need these addresses anymore? Can we remove them if not used anywhere?